subjects/devops/clean_the_list/README.md

```diff
@@ -0,0 +1,32 @@
+## clean_the_list
+
+### Instructions
+
+Create a file `shopping.py` that contains a function `clean_list` which takes in a list of strings and performs the following operations on each list item:
```

Comment:
Usually the delivered file has the same name of the exercise. Maybe some automations and scripts will need that in order to work. Do you think it's ok to change `shopping.py` into `clean_the_list.py`?

Reply:
I am following the instructions in Jira, although I see your point, I wouldn't mind to change it. But if you want it to be that way, you could ask why it was instructed that way on the ticket on the first place before asking to change it.